### PR TITLE
Use HTTPS for links to en.wikipedia.org, bug #637190

### PR DESCRIPTION
**Pull request by fturco**

### CODE REVIEW

**Comment by fturco** [comment]
@mgorny: I split this pull request to one commit per package. I think we can merge now...

**Comment by fturco** [comment]
I just removed two conflicting files from my branch.

File: sys-apps/ack/files/ack-2.14-gentoo.patch [hunk 1]
```diff
@@ -1,7 +1,7 @@
 --- ack-2.14/ConfigDefault.pm
 +++ ack-2.14/ConfigDefault.pm
 @@ -252,6 +252,9 @@
- # http://en.wikipedia.org/wiki/Fortran
+ # https://en.wikipedia.org/wiki/Fortran
```
Comment (mgorny) [inline sys-apps/ack/files/ack-2.14-gentoo.patch]: You're changing patch context.

**Comment by mgorny** [comment]
Ok, I've merged as much as I could. Please rebase and see if there's anything left to be done. Otherwise, please close the PR.

**Comment by fturco** [comment]
There were still some HTTP links in `dev-libs/openssl-0.9.8z_p8-r1.ebuild`.

**Pull request closed without merging**